src/tech/v3/dataset/sql.clj

```diff
@@ -528,10 +528,10 @@ _unnamed [2 1]:
   true
 ```
   "
-    [conn dataset]
+    [conn dataset & options]
```

Comment:
I think for older versions of Clojure this should be `& [options]` instead of `& options`.  Else it will be a map wrapped in a sequence.